vclib/src/commonMain/kotlin/at/asitplus/wallet/lib/data/DurationSecondsIntSerializer.kt

```diff
@@ -0,0 +1,24 @@
+package at.asitplus.wallet.lib.data
+
+import kotlinx.serialization.KSerializer
+import kotlinx.serialization.descriptors.PrimitiveKind
+import kotlinx.serialization.descriptors.PrimitiveSerialDescriptor
+import kotlinx.serialization.descriptors.SerialDescriptor
+import kotlinx.serialization.encoding.Decoder
+import kotlinx.serialization.encoding.Encoder
+import kotlin.time.Duration
+
+object DurationSecondsIntSerializer : KSerializer<Duration> {
+
+    override val descriptor: SerialDescriptor =
+        PrimitiveSerialDescriptor("DurationSecondsIntSerializer", PrimitiveKind.LONG)
+
+    override fun deserialize(decoder: Decoder): Duration {
+        return Duration.parse("${decoder.decodeInt()}s")
```

Comment:
I know it is functionally equivalent, but wouldn't `decoder.decodeInt().seconds` be more concise, straight-forward and safe a virtually useless string instantiation?